syntax = "proto3";

package Protocol;

message LOGIN_RES{
	int32 result = 1;
	string message = 2;

}